shared/src/main/kotlin/com/egm/stellio/shared/service/EntitiesEventService.kt

```diff
@@ -0,0 +1,36 @@
+package com.egm.stellio.shared.service
+
+import com.egm.stellio.shared.model.EntitiesEvent
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.cloud.stream.binding.BinderAwareChannelResolver
+import org.springframework.messaging.MessageHeaders
+import org.springframework.messaging.support.MessageBuilder
+import org.springframework.scheduling.annotation.Async
+import org.springframework.stereotype.Component
+
+@Component
+class EntitiesEventService(
```

Comment:
I don't really get why this service is in the shared module. Currently, it is specific to entity service module.

Reply:
moved to entity-service, are we planning to create separate listeners in other services (an EntitiesEventListenerService in subscription and search)  ??